jetty-quic/quic-client/src/main/java/org/eclipse/jetty/quic/client/ClientQuicConnection.java

```diff
@@ -80,7 +80,10 @@ public void onOpen()
             QuicheConfig quicheConfig = new QuicheConfig();
             quicheConfig.setApplicationProtos(protocols.toArray(String[]::new));
             quicheConfig.setDisableActiveMigration(quicConfiguration.isDisableActiveMigration());
-            quicheConfig.setVerifyPeer(quicConfiguration.isVerifyPeerCertificates());
+            quicheConfig.setVerifyPeer(!connector.getSslContextFactory().isTrustAll());
```

Comment:
Because `verifyPeerCertificates` is not a QUIC setting but a Quiche one. How certificates are configured at the QUIC level is via the standard `SslContextFactory`.